apps/web/src/components/GetStarted/BuildersMostWanted.tsx

```diff
@@ -0,0 +1,91 @@
+import classNames from 'classnames';
+import { sectionContainerClasses, gridClasses } from '../../../app/(base-org)/getstarted/page';
+import { Icon } from '../Icon/Icon';
+import { Resource } from './resourceTypes';
+import ResourceCard from './ResourceGrid/ResourceCard';
+import { FUNDING_SECTION_ID } from './Funding';
+import { GET_NOTICED_SECTION_ID } from './GetNoticed';
+import { START_BUILDING_SECTION_ID } from './StartBuilding';
+
+type MostWantedResource = Resource & {
+  icon: string;
+};
+
+const mostWanted: MostWantedResource[] = [
```

Comment:
Admittedly this is my fault for introducing this pattern, sorry. https://gist.github.com/kevmo314/7bbf5d5868b973b53898d12b977b15b7

Reply:
interesting article, thanks for sending. I can fix if we prefer the non-array method

Reply:
converted all components to explicit declarations instead of arrays / array-mapping